support-workers/src/main/scala/com/gu/emailservices/DigitalPackEmailFields.scala

```diff
@@ -144,19 +144,7 @@ class DigitalPackEmailFields(
   )
 
   def build(digi: SendThankYouEmailDigitalSubscriptionState)(implicit ec: ExecutionContext): Future[List[EmailFields]] =
```

Comment:
don't think this needs to be a list any more, at least at the time, this was the only product that sent multiple emails 
```suggestion
  def build(digi: SendThankYouEmailDigitalSubscriptionState)(implicit ec: ExecutionContext): Future[EmailFields] =
```